pkg/smokescreen/acl/v1/yaml_loader.go

```diff
@@ -78,10 +86,25 @@ func (cfg *YAMLConfig) Load() (*ACL, error) {
 			return nil, err
 		}
 
+		var allowedHostsMitm []MitmDomain
+
+		for _, w := range v.AllowedHostsMitm {
+			mitmDomain := MitmDomain{
```

Comment:
Since this is used a few times I wonder if it could use a `NewMITMDomain()` function as an initializer?